modules/configuring-the-training-operator-permissions-without-kueue.adoc

```diff
@@ -0,0 +1,157 @@
+:_module-type: PROCEDURE
+
+[id="configuring-the-training-operator-permissions-without-kueue_{context}"]
+= Configuring the Training Operator permissions without Kueue
+
+[role='_abstract']
+If you want to run a PyTorch training job in an environment where Kueue is not installed, you must provide the necessary access permissions to a service account.
+
+If you use Kueue to run PyTorch jobs, you can skip this section. 
+
+
+.Prerequisites
+ifdef::upstream,self-managed[]
+* You have logged in to {openshift-platform} with the `cluster-admin` role.
+endif::[]
+ifdef::cloud-service[]
+* You have logged in to OpenShift with the `cluster-admin` role.
+endif::[]
+
+ifndef::upstream[]
+* You have access to a data science cluster that is configured to run distributed workloads as described in link:{rhoaidocshome}{default-format-url}/working_with_distributed_workloads/configuring-distributed-workloads_distributed-workloads[Configuring distributed workloads].
+endif::[]
+ifdef::upstream[]
+* You have access to a data science cluster that is configured to run distributed workloads as described in link:{odhdocshome}/working-with-distributed-workloads/#configuring-distributed-workloads_distributed-workloads[Configuring distributed workloads].
+endif::[]
+
+ifndef::upstream[]
+* You have created a data science project. 
+For information about how to create a project, see link:{rhoaidocshome}{default-format-url}/working_on_data_science_projects/working-on-data-science-projects_nb-server#creating-a-data-science-project_nb-server[Creating a data science project].
+endif::[]
+ifdef::upstream[]
+* You have created a data science project. 
+For information about how to create a project, see link:{odhdocshome}/working-on-data-science-projects/#_using_data_science_projects[Creating a data science project].
```

Comment:
adjust link to be:
For information about how to create a project, see link:{odhdocshome}/working-on-data-science-projects/#creating-a-data-science-project_projects[Creating a data science project]. 

FYI I corrected it in other distributed workload files, such as configuring-quota-management-for-distributed-workloads.adoc, in an already merged MR.